### PR TITLE
[SCB-679] Support Cross-Origin Resource Sharing (CORS)

### PR DESCRIPTION
**Pull request by yhs0092**
Follow this checklist to help us incorporate your contribution quickly and easily:

 - [x] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/SCB) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
 - [x] Each commit in the pull request should have a meaningful subject line and body.
 - [x] Format the pull request title like `[SCB-XXX] Fixes bug in ApproximateQuantiles`, where you replace `SCB-XXX` with the appropriate JIRA issue.
 - [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
 - [x] Run `mvn clean install` to make sure basic checks pass. A more thorough check will be performed on your pull request automatically.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

---
See details in [SCB-679](https://issues.apache.org/jira/browse/SCB-679)

### CODE REVIEW

**Comment by yhs0092** [comment]
Wait for a sec, I need to add some fix in this pr.

**Comment by yhs0092** [comment]
Done. Please review.
- add alias to default executor thread pool, let users can use "servicecomb.executor.groupThreadPool" to specify executor thread poll.
- add warning log to remind users of schema compatibility problem.

File: demo/demo-crossapp/crossapp-server/src/main/resources/microservice.yaml [hunk 1]
```diff
@@ -31,3 +31,10 @@ servicecomb:
       address: http://127.0.0.1:30100
   rest:
     address: 127.0.0.1:8080
+  cors:
```
Comment (mt-monster) [inline demo/demo-crossapp/crossapp-server/src/main/resources/microservice.yaml]: cros?

File: service-registry/src/main/java/org/apache/servicecomb/serviceregistry/task/MicroserviceRegisterTask.java [hunk 2]
```diff
@@ -307,7 +307,10 @@ private void checkRemainingSchema(Map<String, GetSchemaResponse> scSchemaMap) {
 
       // Currently nothing to do but print a warning
       LOGGER.warn("There are schemas only existing in service center: {}, which means there are interfaces changed. "
-          + "It's recommended to increment microservice version before deploying.", scSchemaMap.keySet());
+              + "It's recommended to increment microservice version before deploying.",
+          scSchemaMap.keySet());
+      LOGGER.warn("ATTENTION: The schemas in new version are less than the old version, "
```
Comment (liubao68) [inline service-registry/src/main/java/org/apache/servicecomb/serviceregistry/task/MicroserviceRegisterTask.java]: I think this warning message is the same as above one. Anyway, it's fine to keep it.

File: demo/demo-crossapp/crossapp-server/src/main/resources/microservice.yaml (hunk 1 above)
Comment (yhs0092) [reply demo/demo-crossapp/crossapp-server/src/main/resources/microservice.yaml]: Support Cross-Origin Resource Sharing (CORS)

File: service-registry/src/main/java/org/apache/servicecomb/serviceregistry/task/MicroserviceRegisterTask.java (hunk 2 above)
Comment (yhs0092) [reply service-registry/src/main/java/org/apache/servicecomb/serviceregistry/task/MicroserviceRegisterTask.java]: This is for the review opinion in another [PR-757](https://github.com/apache/incubator-servicecomb-java-chassis/pull/757). I need to warn users to take care of compatibility issues.
I don't know how to describe it clearer with my poor English  :P

File: core/src/main/resources/META-INF/spring/cse.bean.xml [hunk 3]
```diff
@@ -36,5 +36,6 @@
 
   <bean id="cse.executor.groupThreadPool" class="org.apache.servicecomb.core.executor.FixedThreadExecutor"></bean>
   <alias name="cse.executor.groupThreadPool" alias="cse.executor.default"/>
+  <alias name="cse.executor.groupThreadPool" alias="servicecomb.executor.groupThreadPool"/>
   <bean id="cse.executor.reactive" class="org.apache.servicecomb.core.executor.ReactiveExecutor"></bean>
```
Comment (wujimin) [inline core/src/main/resources/META-INF/spring/cse.bean.xml]: cse.executor.reactive also need a alias?

File: demo/demo-crossapp/crossapp-client/src/main/java/org/apache/servicecomb/demo/crossapp/CrossappClient.java [hunk 4]
```diff
@@ -32,12 +40,14 @@
   private static HelloWorld helloWorld;
 
   public static void main(String[] args) throws Exception {
+    System.setProperty("sun.net.http.allowRestrictedHeaders", "true");
```
Comment (wujimin) [inline demo/demo-crossapp/crossapp-client/src/main/java/org/apache/servicecomb/demo/crossapp/CrossappClient.java]: a little strange
crossapp demo is to verify cross ServiceComb application

but no problem, we will merge most demo and integration test to one or two project in the future.

File: core/src/main/resources/META-INF/spring/cse.bean.xml (hunk 3 above)
Comment (yhs0092) [reply core/src/main/resources/META-INF/spring/cse.bean.xml]: an alias has been added for cse.executor.reactive

**Pull request merged**